docs/pattern/form.en-US.md

```diff
@@ -0,0 +1,170 @@
+---
+order: 2
+title:
+  zh-CN: 表单
+  en-US: Form
+---
+
+A form is an important human machine interface to obtain user information. It's necessary to consider what questions should be asked according to the expected answers.
+
+In the design of a form, it's recommended to:
+
+1. Let a user knows exactly what information is required and why he needs to provide this information.
```

Comment:
I think rest of the translation can also be improved this way, by making it less verbose. However, it's just my personal opinion anyway.